projects/ngx-quill/src/lib/quill-editor.component.ts

```diff
@@ -613,3 +595,50 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
     return valid ? null : err
   }
 }
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  providers: [
+    {
+      multi: true,
+      provide: NG_VALUE_ACCESSOR,
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
+      useExisting: forwardRef(() => QuillEditorComponent)
+    },
+    {
+      multi: true,
+      provide: NG_VALIDATORS,
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
+      useExisting: forwardRef(() => QuillEditorComponent)
+    }
+  ],
+  selector: 'quill-editor',
+  template: `
+  <ng-content select="[quill-editor-toolbar]"></ng-content>
+`
+})
+export class QuillEditorComponent extends QuillEditorBase {
+
+  constructor(
```

Comment:
This constructor signature is identical to the original `QuillEditorComponent`, with the addition of a `super()` call.